source/content/guides/support/02-contact-support.md

```diff
@@ -71,12 +69,12 @@ You can download a transcript of your chat with the <Icon icon="arrowDownToLine"
 
 <Alert title="Note" type="info">
 
-Some in-browser ad or tracking blockers can interfere with the **Launch Chat** button. If you don't see it, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has posted more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
+Some in-browser ad or tracking blockers may interfere with the **Start chat** button. If you don't see the button, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has provided more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
```

Comment:
Keep the phrase "Launch Chat" here (Hermes), but the rest of the changes are good.

Reply:
Let's standardize all pages to "Start chat" as it's more direct and understandable for most users. It clearly conveys the action the user is about to take, which is to begin a conversation.

Reply:
Duplicate feedback, marking as resolved to be consolidated in [the above discussion](https://github.com/pantheon-systems/documentation/pull/9191/files#r1729305038)